app/src/components/main/analytics/events/stepPageEvents.js

```diff
@@ -29,6 +29,8 @@ import {
   getClickIssueTicketEvent,
   getClickUniqueErrorsEvent,
   getClickExpandStackTraceArrowEvent,
+  getIgnoreBtnIgnoreItemsInAAModal,
+  getIncludeBtnIncludeInAAModal,
```

Comment:
Please add `Event` postfix for these functions, f.e. getIncludeBtnIncludeInAAModal**Event**